routes/thoughtRoutes.js

```diff
@@ -0,0 +1,75 @@
+import express from "express";
+import listEndpoints from "express-list-endpoints";
+import { ThoughtModel } from "../models/ThoughtModel";
+
+const router = express.Router();
+
+// List of all endpoints
+router.get("/", async (req, res) => {
+  try {
+    res.status(200).send({
+      success: true,
+      message: "OK",
+      body: {
+        content: "Susannes Happy thoughts API",
+        endpoints: listEndpoints(router),
```

Comment:
Adding a body with a content other than the endpoints to your startpage is a really nice addition to the API! Makes the user understand where they have ended up - great job!